app/static/styles/main.css

```diff
@@ -1,5 +1,17 @@
-p, h1, h2, h3, h4, h5, h6, button {
+p, h2, h3, h4, h5, h6, button {
     font-family: "Roboto", serif;
+    margin-right: 10px;
+    margin-left: 15px;
+}
+
+h1 {
```

Comment:
Maybe modifying the h1 only for your section is usually a best practice so as not to affect global styles